py/kubeflow/testing/util.py

```diff
@@ -715,6 +717,16 @@ def split_gcs_uri(gcs_uri):
   return bucket, path
 
 
+def split_s3_uri(s3_uri):
+  """Split a S3 URI into bucket and path."""
```

Comment:
the codes are exact same. You'd better have a common method called `split_object_storage_uri`.